holoviews/plotting/plot.py

```diff
@@ -597,6 +613,32 @@ class GenericElementPlot(DimensionedPlot):
     logy = param.Boolean(default=False, doc="""
         Whether the y-axis of the plot will be a log axis.""")
 
+    padding = param.ClassSelector(default=0, class_=(int, float, tuple), doc="""
```

Comment:
Personally, I would just specify float and not make special allowances for int...

Reply:
So you would want a user specifying ``0`` to raise an error?

Reply:
Since you haven't replied I'm going to leave this, I don't think ``0`` should be an error.

Reply:
I don't think 0 should be an error either.